### ISSUE TITLE
Unresolved reference

### ISSUE THREAD

**Issue by mashail** [bazelbuild/intellij#3328]
Hello,

Our project is a golang monorepo built with bazel.  The tree of the project looks like this

```
.
├── BUILD.bazel
├── README.md
├── Review.md
├── Taskfile.yml
├── WORKSPACE
├── build.sh
├── download.bzl
├── go.mod
├── go.sum
├── k8s
├── k8s-roach
├── nogo.json
├── packages
│   ├── BUILD.bazel
│   ├── customers
....
│   ├── proto
│   │   ├── accounts
│   │   │   └── v1
│   │   │       ├── BUILD.bazel
│   │   │       ├── accounts.proto
│   │   │       ├── client.go **
│   │   │       ├── tx.go **
│   │   │       └── tx_test.go **
......
├── repos.bzl
├── scripts

```

Here is the BUILD file under proto/accounts/v1
```
load("@rules_proto//proto:defs.bzl", "proto_library")
load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")

proto_library(
    name = "cigar_accounts_v1_proto",
    srcs = ["accounts.proto"],
    visibility = ["//visibility:public"],
    deps = [
        "//packages/proto/common/v1:cigar_common_v1_proto",
        "@com_github_envoyproxy_protoc_gen_validate//validate:validate_proto",
        "@com_google_protobuf//:duration_proto",
        "@com_google_protobuf//:timestamp_proto",
    ],
)

go_proto_library(
    name = "cigar_accounts_v1_go_proto",
    compilers = [
        "@io_bazel_rules_go//proto:go_grpc",
        "//bazel:pgv_plugin_go",
    ],
    importpath = "cigar/genproto/accounts/v1",
    proto = ":cigar_accounts_v1_proto",
    visibility = ["//visibility:public"],
    deps = [
        "//packages/proto/common/v1:common",
        "@com_github_envoyproxy_protoc_gen_validate//validate:go_default_library",
    ],
)

go_library(
    name = "accounts",
    srcs = [
        "client.go",
        "tx.go",
    ],
    embed = [":cigar_accounts_v1_go_proto"],  # keep
    importpath = "cigar/genproto/accounts/v1",  # keep
    visibility = ["//visibility:public"],
    deps = ["@org_golang_google_grpc//:go_default_library"],
)

go_test(
    name = "accounts_test",
    srcs = ["tx_test.go"],
    embed = [":accounts"],
    deps = ["@com_github_stretchr_testify//assert"],
)

```

Once we added the *.go file the proto/accounts/v1 everything under this package even the generated proto code became unresolved in Intellij IDEA.

before adding the *.go under  proto/accounts/v1  it was resolving them without any issue. Any idea what should I change to fix this

**Comment by mashail** [comment]
help please

**Comment by tingilee** [comment]
+1 Also seeing this issue.

**Comment by blorente** [comment]
This looks like it will be fixed by @tingilee 's contribution here: https://github.com/bazelbuild/intellij/pull/6030

**Comment by tpasternak** [comment]
Probably fixed #6030
